### PR TITLE
Feat: Creates note component for CRM

### PR DESCRIPTION
**Pull request by iamsivin**
Screenshot 2021-04-29 at 6 02 11 PM

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/contacts/components/ContactNote.vue
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/contacts/components/ContactNote.vue:115]: We might need to add small icon button to WootButton

**Review by nithindavid — APPROVED** [review]
Looks good. Will merge once storybook PR is merged.
Also if possible add small icon button styles to WootButton in a new PR.

File: app/javascript/dashboard/routes/dashboard/contacts/components/ContactNote.vue
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/contacts/components/ContactNote.vue:21]: Can we use appropriate  props for woot-button?

**Review by nithindavid — CHANGES_REQUESTED** [review]

**Review by nithindavid — APPROVED** [review]

**Comment by muhsin-k** [comment]
@iamsivin Good work 👏🏻. I have a small suggestion here if you can make the story file name in the format of `[component_name].stories.js`. It will be easy and helpful to find the stories easily

**Comment by iamsivin** [comment]
> @iamsivin Good work 👏🏻. I have a small suggestion here if you can make the story file name in the format of `[component_name].stories.js`. It will be easy and helpful to find the stories easily

Yeah sure.

**Pull request merged**
